src/prom-metrics/definitions/fuelTank.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import client from 'prom-client'
+import register from '../registry'
+import connection from '../../connection'
+
+export const indexer_fueltanks_tanks_total = new client.Gauge({
+    name: 'indexer_fueltanks_tanks_total',
+    labelNames: ['fueltank'],
+    help: 'The total number of fuel tanks.',
+    registers: [register],
+})
+
+export const indexer_fueltanks_calls_dispatched_total = new client.Gauge({
+    name: 'indexer_fueltanks_calls_dispatched_total',
+    labelNames: ['fueltank'],
+    help: 'The total number of calls dispatched using a fuel tank.',
+    registers: [register],
+})
+
+export const indexer_fueltanks_enj_consumed_total = new client.Gauge({
+    name: 'indexer_fueltanks_enj_consumed_total',
+    labelNames: ['fueltank'],
+    help: 'The total amount of ENJ consumed through the use of fuel tanks.',
+    registers: [register],
+})
+
+export const indexer_fueltanks_enj_subsidy_avg = new client.Gauge({
+    name: 'indexer_fueltanks_enj_subsidy_avg',
+    labelNames: ['fueltank'],
+    help: 'The average amount of ENJ that’s subsidized by a fuel tank.',
+    registers: [register],
```

Comment:
Just wondering if we need to prefix everything with `indexer_` are there other system that the data is also collected?

Reply:
i am not sure, but these variable names are mentioned in the docs by Brad. and i used them as they are written.

Reply:
If that's the case, it is already approved, go for it.